opensrp-app/src/test/java/org/smartregister/CoreLibraryTest.java

```diff
@@ -192,4 +197,40 @@ public void constructorShouldInitialisePathEvaluatorLibrary() {
         Assert.assertNotNull(PathEvaluatorLibrary.getInstance().getTaskProvider().getTaskDao());
         Assert.assertNotNull(PathEvaluatorLibrary.getInstance().getEventProvider().getEventDao());
     }
+
+    @Test
+    public void copySharedPreferences() {
```

Comment:
It would be very useful if the test method name was a little bit more descriptive like some of the others committed in this PR e.g. `creatingUserWithNullShouldThrowException`

Reply:
This method has been updated